website/status.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: home
+title:  "System Status"
+date:   2019-11-02 09:21:12 -0500 
+description: This page gives an overview of the status of our systems.
+landing-page: live
+gradient: "blueberry-lime-background"
+subnav-link-gradient: "blueberry-lime-link"
+sections:
+  - name: Home
+    link: /
+ctas:
+
+---
+
+<script>
+    let baseUrl = 'https://sandbox.ab2d.cms.gov/';
+
+    function pollServer() {
+        $.get(baseUrl + 'status', function(data) {
+            if(data.maintenanceMode === 'false') {
+                $('#status-content').html("<img style=\"vertical-align: middle;\" src='assets/img/status-up.png' /> <span>The system is operating normally</span>");                     
```

Comment:
Just used a check and x since I couldn't find a thumbs up/down that looked good.